atlassian_jwt_auth/key.py

```diff
@@ -64,32 +66,34 @@ def __init__(self, base_url):
         if not base_url.endswith('/'):
             base_url += '/'
         self.base_url = base_url
-        self._session = None
+        self._session = self._get_session()
```

Comment:
It does not look like a major bump, major should happen if there's a backwards incompatible change to a public API. `_` attributes are not public, they are rather private. Is it a significant behavior change?